documents/registernach.md

```diff
@@ -47,8 +47,19 @@ $api->order->create(array('amount' => 100, 'currency' => 'INR', 'method'=>'nach'
 | currency*   | string  | The currency of the payment (defaults to INR)  |
 | customerId*   | string      | The id of the customer to be fetched |
 | method*      | string  | Payment method used to make the registration transaction. Possible value is `nach`.  |
-| receipt      | string  | Your system order reference id.  |
-| token  | array  | All keys listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/auto-debit/#112-create-an-order) are supported |
+| token["first_payment_amount"]  | integer  |  The amount, in paise, the customer should be auto-charged in addition to the authorization amount. |
```

Comment:
-         All keys listed here are supported for customer

Reply:
added list link like before